application/controllers/register.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+require "./vendor/autoload.php";
+
+$validate = new Validation();
+$database = new Database();
+$pathOfProfilePhoto = "";
```

Comment:
Class variables are camelCased.
Non-class variables are snake cased.
So this should be: `$path_of_profile_photo`.